pandas_gbq/tests/test_gbq.py

```diff
@@ -83,100 +83,7 @@ def _get_private_key_contents():
         return f.read()
 
 
-def _test_imports():
```

Comment:
pls don't remove things like this. just change to use the new deps. This is going to hard fail if deps are not installed.